map/map_loader/README.md

```diff
@@ -138,12 +139,16 @@ Please see [the description of `GetSelectedPointCloudMap.srv`](https://github.co
 ### Feature
 
 lanelet2_map_loader loads Lanelet2 file and publishes the map data as autoware_auto_mapping_msgs/HADMapBin message.
-The node projects lan/lon coordinates into MGRS coordinates.
+The node projects lan/lon coordinates into arbitrary coordinates defined in `/map/map_projector_type` from `map_projection_loader`.
 
```

Comment:
```suggestion
The node projects lan/lon coordinates into arbitrary coordinates defined in `/map/map_projector_type` from `map_projection_loader`.
The node supports the following four types of coordinate systems:

- MGRS
- UTM
- TransverseMercator
- local
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3986/commits/4b8ddbbb0bd9f0870cd822993b0d3fc44c866fbe